### ISSUE TITLE
Mobile signal not always appear after reboot

### ISSUE THREAD

**Issue by ngoquang2708** [ngoquang2708/android_device_samsung_vivalto3gvn#105]
It will appear if I restart/kill ```rild```s.

**Comment by ngoquang2708** [comment]
@remilia15 Does your device have this issue?

**Comment by diepquynh** [comment]
No, but RIL takes about a minute to get dual SIM signals, and single SIM is immediately after successfully booted

**Comment by ngoquang2708** [comment]
In my case, everything is OK, rild did not crash, got imei, just empty signal bar.

**Comment by ih24n69** [comment]
@ngoquang2708 @diepquynh1501 I have same problem got imei, got baseband, no ril crash but my signalbar still 0 . what you do to make this issue solve?
